component-api/src/main/java/org/talend/sdk/component/api/service/discovery/DiscoverDatasetResult.java

```diff
@@ -48,6 +48,7 @@ public DatasetDescription(final String name) {
         public void addMetadata(final String key, final String value) {
             this.metadata.put(key, value);
         }
+
```

Comment:
extra line break by formater?

Reply:
I had added methods in this class, then removed them as they appeared to be useless. I may have kept a break.
Corrected.